test_bitmap_to_screen_adapter fails on main (windows, py 3.12)

```
========================================================================================================================================================= FAILURES ========================================================================================================================================================= 
______________________________________________________________________________________________________________________________________________ test_bitmap_to_screen_adapter _______________________________________________________________________________________________________________________________________________ 

    @pytest.mark.skipif(not can_use_wgpu_lib, reason="Needs wgpu lib")
    def test_bitmap_to_screen_adapter():
        # Create canvas and attach our special adapter canvas
        canvas = ManualOffscreenRenderCanvas()
        context = canvas.get_context(__name__ + ":SpecialContextWithWgpuAdapter")

        # Create and set bitmap
        bitmap = get_test_bitmap(*canvas.get_physical_size())
        context.set_bitmap(bitmap.copy())

        # Draw! This will call SpecialContextWithWgpuAdapter.present(), which will
        # invoke the adapter to render the bitmap to a texture. The GpuCanvasContext.present()
        # method will also be called, which will download the texture to a bitmap,
        # and that's what we receive as the result.
        # So this little line here touches quite a lot of code. In the end, the bitmap
        # should be unchanged, because the adapter assumes that the incoming bitmap
        # is in the sRGB colorspace.
        result = canvas.draw()

        assert isinstance(result, memoryview)
        result = np.asarray(result)
>       assert np.all(result == bitmap)
E       assert np.False_
E        +  where np.False_ = <function all at 0x000002475ED20970>(array([[[255,..., dtype=uint8) == array([[[255,..., dtype=uint8)
E        +    where <function all at 0x000002475ED20970> = np.all
E           
E           Use -v to get more diff)

tests\test_context.py:233: AssertionError
```

--- a/rendercanvas/utils/bitmappresentadapter.py
+++ b/rendercanvas/utils/bitmappresentadapter.py
@@ -10,7 +10,7 @@ class BitmapPresentAdapter:
     """An adapter to present a bitmap to a canvas using wgpu.
 
     This adapter can be used by context objects that want to present a bitmap, when the
-    canvas only supoorts presenting to screen.
+    canvas only supports presenting to screen.
     """
 
     def __init__(self, canvas, present_methods):
@@ -37,6 +37,10 @@ class BitmapPresentAdapter:
 
         if not self._context_is_configured:
             format = self._context.get_preferred_format(self._device.adapter)
+            # We don't want an srgb texture, because we assume the input bitmap is already srgb.
+            # AFAIK contexts always support both the regular and the srgb texture format variants
+            if format.endswith("-srgb"):
+                format = format[:-5]
             self._context.configure(device=self._device, format=format)
 
         target = self._context.get_current_texture().create_view()

--- a/tests/test_context.py
+++ b/tests/test_context.py
@@ -210,7 +210,7 @@ def test_bitmap_context():
 
 
 @pytest.mark.skipif(not can_use_wgpu_lib, reason="Needs wgpu lib")
-def test_bitmap_to_screen_adapter():
+def test_bitmap_present_adapter():
     # Create canvas and attach our special adapter canvas
     canvas = ManualOffscreenRenderCanvas()
     context = canvas.get_context(__name__ + ":SpecialContextWithWgpuAdapter")

